ndmg/register/gen_reg.py

```diff
@@ -886,9 +886,10 @@ def dwi2atlas(self, clean=False):
 
         # Applies skull stripping to T1 volume, then EPI alignment to T1
         print(
-            "calling mgru.extract_brain on {}, {}".format(self.t1w, self.t1w_brain)
+            "calling extract_brain on {}, {}".format(self.t1w, self.t1w_brain)
         )  # t1w = in, t1w_brain = out
-        mgru.extract_brain(self.t1w, self.t1w_brain, "-B")
+        mgru.t1w_skullstrip(self.t1w, self.t1w_brain)
```

Comment:
Reason for the change?
looks like it's using Afni's 3dSkullStrip now instead of bet?

Reply:
yes, because BET was breaking on some images at random. 3dSkullStrip is a bit of enhancement to greg's old procedure, but it also makes for a fairer comparison of the old MNI-space method to the new native-space routines.